bundle/manifests/apicast-operator.clusterserviceversion.yaml

```diff
@@ -28,6 +28,10 @@ metadata:
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v2
     repository: https://github.com/3scale/apicast-operator
     support: Red Hat
+  labels:
```

Comment:
changes needs to be done in the config/manifests/bases/apicast-operator.clusterserviceversion.yaml

Reply:
then run `make bundle`

Reply:
this is done, resolving this conversation

Reply:
I do not see any change

Reply:
The changes should be visible now :)